ci/buildkite-pipeline.sh

```diff
@@ -305,8 +305,12 @@ EOF
     ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-flakey.sh" \
     10
 
-  command_step "local-cluster-slow" \
-    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow.sh" \
+  command_step "local-cluster-slow-1" \
+    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow-1.sh" \
+    40
+
+  command_step "local-cluster-slow-2" \
+    ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_stable_docker_image ci/test-local-cluster-slow-2.sh" \
```

Comment:
what was the motivation for splitting the job instead of just extending the first slow job?  This is going to consume an extra CI agent

Reply:
Trying to reduce the pain of having to retry a failed single big local-cluster-slow job. Typing that out makes me think maybe those tests should go into flakey instead. Want me to smoosh these back together into a single slow job?

Reply:
I one would be better for CI.  Agreed that if a test needs retried for failure (not timeout) it should be in flaky.  If someone deflakes and it's still slow, it can go to slow

Reply:
I'm personally of the opinion that using more CI agents for shorter periods of time is better for development speed and progress. I know that setting up each job has a non-zero compute cost, but worth the trade off imho